scripts/actions/spells/trust/makki-chebukki.lua

```diff
@@ -13,7 +13,33 @@ end
 
 spellObject.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
```

Comment:
This trust has chat lines: https://ffxiclopedia.fandom.com/wiki/Trust:_Makki-Chebukki